content/kb/tutorials/databases/snowflake.md

```diff
@@ -148,66 +145,11 @@ for row in df.itertuples():
     st.write(f"{row.NAME} has a :{row.PET}:")
 ```
 
-This example uses `with conn.safe_session()` to provide thread safety. `conn.session` also works directly, but does not guarantee thread safety. If everything worked out (and you used the example table we created above), your app should look the same as the screenshot from the first example above.
-
-## Using the Snowflake Connector for Python
-
-In some cases, you may prefer to use the [Snowflake Connector for Python](https://docs.snowflake.com/en/developer-guide/python-connector/python-connector) instead of Snowpark Python. Streamlit supports this natively through the [SQLConnection](/library/api-reference/connections/st.connections.sqlconnection) and the [snowflake-sqlalchemy](https://docs.snowflake.com/en/developer-guide/python-connector/sqlalchemy) library.
-
-```bash
-pip install snowflake-sqlalchemy
-```
-
-Installing `snowflake-sqlalchemy` will also install all necessary dependencies.
-
-Configuring credentials follows the `SQLConnection` format which is slightly different. See the Snowflake SQLAlchemy [Configuration Parameters](https://docs.snowflake.com/en/developer-guide/python-connector/sqlalchemy#connection-parameters) documentation for more details.
-
-```toml
-# .streamlit/secrets.toml
-
-[connections.snowflake]
-url = "snowflake://<user_login_name>:<password>@<account_identifier>/<database_name>/<schema_name>?warehouse=<warehouse_name>&role=<role_name>"
-```
-
-Alternatively, specify connection parameters like `authenticator` or key pair authentication using `create_engine_kwargs`, as shown below.
-
-```toml
-# .streamlit/secrets.toml
-
-[connections.snowflake]
-url = "snowflake://<user_login_name>@<account_identifier>/"
-
-[connections.snowflake.create_engine_kwargs.connect_args]
-authenticator = "externalbrowser"
-warehouse = "xxx"
-role = "xxx"
-client_session_keep_alive = true
-```
-
-Initializing and using the connection in your app is similar. Note that [SQLConnection.query()](/library/api-reference/connections/st.connections.sqlconnection#sqlconnectionquery) supports extra arguments like `params` and `chunksize` which may be useful for more advanced apps.
-
-```python
-# streamlit_app.py
-
-import streamlit as st
-
-# Initialize connection.
-conn = st.experimental_connection('snowflake', type='sql')
-
-# Perform query.
-df = conn.query('SELECT * from mytable;', ttl=600)
-
-# Print results.
-for row in df.itertuples():
-    st.write(f"{row.name} has a :{row.pet}:")
-```
-
-If everything worked out (and you used the example table we created above), your app should look the same as the screenshot from the first example above.
+This example uses `with conn.session()` to provide thread safety. `conn.session` also works directly, but does not guarantee thread safety. If everything worked out (and you used the example table we created above), your app should look the same as the screenshot from the first example above.
```

Comment:
I don't think `conn.session` property works any more. I would just remove this line.

Reply:
Will do.

Reply:
Done in [6f5c73b](https://github.com/streamlit/docs/pull/860/commits/6f5c73b9affdee566e3a4d81f05cac6992deb3e5)